cloudinit/version.py

```diff
@@ -4,7 +4,7 @@
 #
 # This file is part of cloud-init. See LICENSE file for license information.
 
-__VERSION__ = "23.2.2"
+__VERSION__ = "23.3.2"
```

Comment:
?
```suggestion
__VERSION__ = "23.3"
```

Reply:
Whoops pushed the branch before I applied my fix from https://github.com/canonical/uss-tableflip/pull/112. Thanks Chris